pkg/manager/member/tikv_member_manager.go

```diff
@@ -433,6 +437,44 @@ func getNewTiKVSetForTidbCluster(tc *v1alpha1.TidbCluster, cm *corev1.ConfigMap)
 		return nil, fmt.Errorf("get delete slots number of statefulset %s/%s failed, err:%v", ns, setName, err)
 	}
 
+	var containers []corev1.Container
+	if tc.Spec.TiDB.ShouldSeparateRocksDBLog() {
+		// mount a shared volume and tail the RocksDB log to STDOUT using a sidecar.
+		rocksDBLogFilePath := path.Join(tikvDataVol.MountPath, "db/LOG")
+		containers = append(containers, corev1.Container{
+			Name:            v1alpha1.RocksDBLogTailerMemberType.String(),
+			Image:           tc.HelperImage(),
+			ImagePullPolicy: tc.HelperImagePullPolicy(),
+			// we can reuse the SlowLogTailerSpec here
+			// or maybe we should rename SlowLogTailerSpec to LogTailerSpec
+			Resources:    controller.ContainerResource(tc.Spec.TiDB.GetSlowLogTailerSpec().ResourceRequirements),
```

Comment:
Please do not use the tailer spec for TiDB, What about adding logTailer 
 for TiKV as we have done for TiFlash https://github.com/pingcap/tidb-operator/blob/master/pkg/apis/pingcap/v1alpha1/types.go#L495.